docs/src/examples/Path planning.jl

```diff
@@ -5,7 +5,7 @@ using Test     #src
 #md # [![nbviewer](https://img.shields.io/badge/show-nbviewer-579ACA.svg)](@__NBVIEWER_ROOT_URL__/generated/Path planning.ipynb)
 #
 # This example was borrowed from [1, IX. Examples, A] whose dynamics comes from the model given in [2, Ch. 2.4].
-# This is a **safety problem** for a **continuous system*.
+#       This is a **reachability problem** for a **continuous system**.
```

Comment:
Why the indent ?

Reply:
this is a mistake. For the image, I hope it will work because I had to guess its URL on Dionysos.

Reply:
We'll see :)